src/content/docs/mlops/integrations/openai-integration.mdx

```diff
@@ -4,58 +4,65 @@ metaDescription: "A lightweight tool to monitor your OpenAI workload."
+With New Relic's GPT series application integration, you have the ability to seamlessly monitor all OpenAI queries. By adding just two lines of code, your team can gain access to key performance metrics such as cost, response time, rate limits, error rates, chat history, and sample inputs and outputs. Our GPT integration provides real-time metrics to help businesses optimize usage, reduce costs, and achieve better results.
 
-By adding just two lines of code, users can gain access to key performance metrics such as cost, response time, and sample inputs/outputs. The dashboard also allows users to track total requests, average token/requests, and model names.
-
-With the fully customizable dashboard of New Relic, users can add additional metrics based on their specific business requirements and needs.
-
-Overall, New Relic's GPT integration provides real-time metrics to help businesses optimize usage, reduce costs, and achieve better results.
+This guide will show you how to integrate you OpenAI GPT applications so you can start monitoring your usage.
 
+Once you instrument find OpenAI observability data under **Model Performance** >  **Application**.
 
 <img
   title="OpenAI dashboard"
   alt="OpenAI dashboard"
   src={modelperformancemonitoringOpenAIDashboard}
 />
 
+<CollapserGroup>
+  <Collapser 
+    id="install-with-python" 
+    title="Install with Python"
+  >
 
-## Installation [#installation]
-
-Installation is straightforward and similar to any Python library.
+<Steps>
+<Step>
+### With pip
 
-```python
+```bash
 pip install nr-openai-observability
 ```
-
-## Getting Started [#getting-started]
-
-This guide takes you step by step for everything needed to integrate OpenAI GPT applications and start monitoring your usage.
-
-### Step 1. Set your environment variable [#set-environment-variable]
+</Step>
+<Step>
+### Set your environment variable [#set-environment-variable]
 
 Get your <InlinePopover type="licenseKey" /> (also referenced as `ingest - license`) and set it as environment variable: `NEW_RELIC_LICENSE_KEY`. [Click here](/docs/apis/intro-apis/new-relic-api-keys/#license-key) for more details and instructions.
 
+```bash
+export NEW_RELIC_LICENSE_KEY=<license key>
+```
+
 ```python
 import os
 os.environ["NEW_RELIC_LICENSE_KEY"] = "<license key>"
 ```
 
 `NEW_RELIC_LICENSE_KEY` can also be sent as a parameter at the `monitor.initialization()` call.
 
+If you are reporting data to the New Relic EU region please follow [these](docs/mlops/integrations/openai-integration/#EU-account-users) instructions.
+</Step>
 
-### Step 2. Modify your code [#modify-code]
+<Step>
+### Modify your code [#modify-code]
 
 Add the following two lines to your code:
 
 ```python
 from nr_openai_observability import monitor
 monitor.initialization()
 ```
-
-### Step 3. Add your dashboard [#add-dashboard]
+</Step>
+<Step>
+### Add your dashboard [#add-dashboard]
 
 Follow the instruction to add the dashboard to your New Relic account:
```

Comment:
Same comment as above. What file are they adding this to? A New Relic config file? Their app code?